dockerfiles/README.md

```diff
@@ -13,7 +13,8 @@ Build and Deploy the Container
     sudo docker tag <IMAGE_ID> <PATH_NAME>:tag . # to tag the image after the build (ex. 0.0.3)
     sudo docker login # login into the account to push to
     sudo docker push <PATH_NAME>:tag # to push to repo with tag
-    sudo docker run -it <IMAGE_ID> bash # to run an interactive version of the container
+    sudo docker run -it --privileged <IMAGE_ID> bash # to run an interactive version of the container
```

Comment:
Why does this need `--privileged` access? Is this to mount the FS for modifications?

Reply:
Yes. I haven't been able to find a way to mount filesystems in docker without --privileged. You'd think FUSE would be fine, but even that doesn't work with the way the kernel handles filesystem access. You can see firesim/firemarshal#199 for the issue tracking this.